test/relations.test.js

```diff
@@ -2152,11 +2250,21 @@ describe('relations', function() {
       Picture.find({include: 'imageable'}, function(err, pics) {
         should.not.exist(err);
         pics.should.have.length(2);
-        pics[0].name.should.equal('Author Pic');
-        pics[0].imageable().name.should.equal('Author 1');
-        pics[1].name.should.equal('Reader Pic');
-        pics[1].imageable().name.should.equal('Reader 1');
-        done();
+        if (isCassandraConnector) {
+          // Sorting is not set in define for Cassandra
+          var names = ['Author Pic', 'Reader Pic'];
+          var imageables = ['Author 1', 'Reader 1'];
+          names.should.containEql(pics[0].name);
+          names.should.containEql(pics[1].name);
+          imageables.should.containEql(pics[0].imageable().name);
+          imageables.should.containEql(pics[1].imageable().name);
+        } else {
+          pics[0].name.should.equal('Author Pic');
+          pics[0].imageable().name.should.equal('Author 1');
+          pics[1].name.should.equal('Reader Pic');
+          pics[1].imageable().name.should.equal('Reader 1');
+        }
```

Comment:
I think one should be able to simplify this long block by using `containDeep`:

```js
const actual = pics.map(p => { 
  name: p.name, 
  imageName: p.imageable().name,
});

actual.should.containDeep([
  {name: 'Author 1', imageName: 'Author Pic' },
  {name: 'Reader 1', imageName: 'Reader Pic' },
]);
```

I think it should be ok to use this single order-agnostic assert for all connectors.